core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/events/SystemNotification.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 B2i Healthcare Pte Ltd, http://b2i.sg
+ * Copyright 2018 B2i Healthcare Pte Ltd, http://b2i.sg
```

Comment:
Indicate start and current year in license header, please, when you modify a file. In this case the correct license year would be `2017-2018`